src/lib/src/viewer/viewer-header/viewer-header.component.html

```diff
@@ -1,16 +1,19 @@
 <mat-toolbar>
-  <div fxLayout='row' fxLayoutAlign="end center" fxFlex>
-    <button id="contentsDialogButton" mat-icon-button [attr.aria-label]="intl.contentsLabel" [matTooltip]="intl.contentsLabel" (click)="toggleContents()">
-      <mat-icon>list</mat-icon>
-    </button>
-    <button id="contentSearchDialogButton" *ngIf="isContentSearchEnabled" mat-icon-button [attr.aria-label]="intl.searchLabel" [matTooltip]="intl.searchLabel" (click)="toggleSearch()">
-      <mat-icon>search</mat-icon>
-    </button>
-    <button id="fullscreenButton" *ngIf="isFullscreenEnabled && !isInFullScreen()" mat-icon-button [attr.aria-label]="intl.fullScreenLabel" [matTooltip]="intl.fullScreenLabel" (click)="toggleFullscreen()">
-      <mat-icon>fullscreen</mat-icon>
-    </button>
-    <button id="exitFullscreenButton" *ngIf="isFullscreenEnabled && isInFullScreen()" mat-icon-button [attr.aria-label]="intl.exitFullScreenLabel" [matTooltip]="intl.exitFullScreenLabel" (click)="toggleFullscreen()">
-      <mat-icon >fullscreen_exit</mat-icon>
-    </button>
+  <div class="header-container" fxLayout="row" fxLayoutAlign="space-between center">
```

Comment:
This div is not necessary. Move label inside current div and add fxFlex to it. Change fxLayoutAlign to start center

Reply:
Your suggestion will not work because of https://github.com/angular/material2/issues/2451
text-overflow: ellipsis does not work well with flex parents and items